kubernetes/elastic-helm/configmap-elastic.yaml

```diff
@@ -21,6 +21,21 @@ data:
     extensions:
     processors:
       batch: {}
+      resource:
+        attributes:
+          - key: deployment.environment
+            value: "opentelemetry-demo"
```

Comment:
Great!

Reply:
Let me create a new PR with just this change for Elastic since this deployment.environment dependency is very Elastic centric.